test/unittest/testAlignment.py

```diff
@@ -8,136 +8,124 @@
 sepp._DEBUG = True
 from sepp.alignment import MutableAlignment, ReadonlySubalignment,\
     ExtendedAlignment
-from sepp.problem import SeppProblem        
+from sepp.problem import SeppProblem
 
 
 
 class Test(unittest.TestCase):
 
     def setUp(self):
         pass
-    
+
     def tearDown(self):
         pass
 
 
     def testAlignmentReadFasta(self):
-        print("====== starting testAlignmentReadFasta ==========") 
```

Comment:
I wanted to keep the console of the CI service clean and thus removed those debugging lines, which are generated by the unit testing system of python anyway.